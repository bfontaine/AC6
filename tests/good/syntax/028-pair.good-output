Line 1, characters 64-65: Error (Parsing)
  Unknown error.

