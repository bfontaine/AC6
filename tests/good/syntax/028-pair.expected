val x : { _0 int * _1 int } =
  { { _0 int * _1 int } _0 <- 42; _1 <- 73 }
